prisma/schema.prisma.example

```diff
@@ -0,0 +1,17 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
```

Comment:
The `DATABASE_URL` will need to be specified in the `.env` file. For connecting to a local instance, I had something like:

```
DATABASE_URL="postgresql://jbranchaud:jbranchaud@localhost:5432/new_egghead_development?schema=public"
```